cropduster/utils/paths.py

```diff
@@ -25,8 +25,8 @@ def get_upload_foldername(file_name, upload_to='%Y/%m'):
     if six.PY2 and isinstance(filename, unicode):
         filename = filename.encode('utf-8')
 
-    root_dir = os.path.splitext(filename)[0]
-    root_dir = dir_name = os.path.join(settings.MEDIA_ROOT, root_dir)
+    image_dir = os.path.splitext(filename)[0]
+    root_dir = dir_name = os.path.join(settings.MEDIA_ROOT, image_dir)
```

Comment:
I think the logic from here to the return might be replaceable by

```python
image_dir = default_storage.get_available_name(dir_name, max_length=255)
```

Reply:
Cool, will test 👍